instances/molecularEntity/DNQX.jsonld

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/molecularEntity/DNQX",
```

Comment:
please follow one convention for chemical entities: I would suggest we always use the IUPAC  name.